File: gradleTransformPlugin/src/main/java/com/doubleyellowice/gradleTransformPlugin/TransformPlugin.java
package com.doubleyellowice.gradleTransformPlugin;

import org.gradle.api.Plugin;
import org.gradle.api.Project;

public class TransformPlugin implements Plugin<Project> {
    @Override
    public void apply(Project target) {
        target.getLogger().lifecycle("TransformPlugin has been applied to project: " + target.getName());
    }
}

File: gradleTransformPlugin/src/main/java/com/doubleyellowice/gradleTransformPlugin/TransformPlugin.kt
package com.doubleyellowice.gradleTransformPlugin

import org.gradle.api.Plugin
import org.gradle.api.Project

class TransformPlugin : Plugin<Project> {

    override fun apply(target: Project) {
        target.logger.lifecycle("TransformPlugin has been applied to project: " + target.name)
    }
}
